### ISSUE TITLE
Mandatory columns in AO

### ISSUE THREAD

**Issue by vinegar-ethz** [gtadigital/grp-userfeedback#8]
AO-tables should show at least a shortened version of actors, places, datings and the description of the holding additionally to identifiers and titles, to allow a quick choice.

**Comment by thomashaensli** [comment]
Good point, however, some of the Archival Objects (namely entire "Konvolute") are linked to dozen's of  actors and places. In order to find a solutions, can you provide an example?

**Comment by vinegar-ethz** [comment]
Take for example #771160 (unfortunately not yet linked to actors, but you see them in the legacy data): take for example the first role («Fachpreisrichter») which is the relevant one together with the first 100 characters of the actors set, then […].
